### PR TITLE
Plugin turbostat

### PR DESCRIPTION
**Pull request by jkroepke**
Added plugin turbostat See: https://collectd.org/documentation/manpages/collectd.conf.5.shtml#plugin_turbostat

### CODE REVIEW

File: manifests/plugin/turbostat.pp [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  $package_c_states            = undef,
+  $system_management_interrupt = undef,
+  $digital_temperature_sensor  = undef,
+  $tcc_activation_temp         = undef,
+  $running_average_power_limit = undef,
+) {
+
+  include ::collectd
+
+  validate_string($core_c_states, $package_c_states, $tcc_activation_temp, $running_average_power_limit)
```
Comment (bastelfreak) [inline manifests/plugin/turbostat.pp]: can you instead declare the variables as String datatype?

File: manifests/plugin/turbostat.pp [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  if $system_management_interrupt {
+    validate_bool($system_management_interrupt)
```
Comment (bastelfreak) [inline manifests/plugin/turbostat.pp]: same here, define as bool please

**Comment by jkroepke** [comment]
@bastelfreak Did you mean the puppet 4 type definitions instead validate functions?

**Comment by bastelfreak** [comment]
yep

**Comment by jkroepke** [comment]
@bastelfreak What about erb? Should I use epp instead erb?

**Comment by bastelfreak** [comment]
@jkroepke can you take a look at this PR as well?

**Comment by jkroepke** [comment]
@bastelfreak sure. I will try it.

**Comment by bastelfreak** [comment]
Thanks!

**Pull request merged**